core-primitives/stf-executor/src/executor.rs

```diff
@@ -125,90 +122,11 @@ where
 	}
 }
 
-impl<OCallApi, StateHandler, ExternalitiesT> StfExecuteTrustedCall
-	for StfExecutor<OCallApi, StateHandler, ExternalitiesT>
+impl<OCallApi, StateHandler> StfUpdateState for StfExecutor<OCallApi, StateHandler>
```

Comment:
Reduce the amount of type parameters for this struct; the `StateHandler` already defines the type of externalities, no need to have another explicit type parameter for that. (removed `ExternalitiesT`).